src/templates/Handbook/Navigation.js

```diff
@@ -17,11 +17,10 @@ export default function Navigation({
     return (
         <div
             style={{
-                transition: 'all 0.5s ease 0s',
```

Comment:
Was this intentional? It removes the navigation slide animation 😄

![recording](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiNzU5MWRkNWEtY2E0My00ZGEzLWI3MGMtYmZkMjIwMzgwODc0IiwiZGVwbG95X2lkIjoiNjEzMzhjM2JmNDdhODEwMDA3NzhmZWZiIiwiaWQiOiI1MmNiMTA1OS0xYWFhLTRhYzItOWZmYS00YTRlNDQxZWNhOWUifQ==)

Reply:
Was not intentional, thanks for catching that!